CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Earthly Changelog
 
-All notable changes to [Earthly](https://github.com/earthly/earthly) will be documented in this file.
+All notable changes to [Earthly](https://github.com/earthbuild/earthbuild) will be documented in this file.
 
 ## Unreleased
 
 ## v0.8.16 - 2025-07-16
 
 ### Changed
-- Removed CLI based account creation [#4283](https://github.com/earthly/earthly/pull/4283)
-- `dockerd` wrapper cleans up after itself [#4267](https://github.com/earthly/earthly/pull/4267)/[#4268](https://github.com/earthly/earthly/pull/4268)
-- Removed all Cloud dependencies, commands, and flags as part of the [Earthly Cloud shutdown](https://earthly.dev/blog/shutting-down-earthfiles-cloud/) [#4235](https://github.com/earthly/earthly/pull/4325)
+- Removed CLI based account creation [#4283](https://github.com/earthbuild/earthbuild/pull/4283)
```

Comment:
the links to old pull requests should stay as `earthly/earthly` otherwise they will be broken, right?